tests/phpunit/Performance/Memory/MemoryFormatterTest.php

```diff
@@ -53,19 +54,33 @@ protected function setUp(): void
     /**
      * @dataProvider bytesProvider
      */
-    public function test_it_converts_bytes_to_string(float $bytes, string $expectedString): void
+    public function test_it_converts_bytes_to_human_readable_time(float $bytes, string $expectedString): void
     {
         $timeString = $this->memoryFormatter->toHumanReadableString($bytes);
 
         $this->assertSame($expectedString, $timeString);
     }
 
-    public function bytesProvider()
+    public function bytesProvider(): Generator
     {
-        yield [1048576, '1.00MB'];
+        yield [-10, '-10.00B'];
```

Comment:
Is it real example? why would we need it? I'm not sure [memory_get_usage()](https://www.php.net/manual/ru/function.memory-get-usage.php) can return negative numbers

Reply:
not a real need but there is also not a need to guard against it... so at least it captures the behaviour

Reply:
Actually we do adapt the code so instead we might just guard against this case. Updated